### ISSUE TITLE
Knob does not show value when value is 0

### ISSUE THREAD

**Issue by AndrewMHenry** [zauberzeug/nicegui#791]
### Description

# Summary

In my NiceGUI project, I have a knob with the default value range of [0, 1], and the value does not show in the center (even with `show_value = True`) if the value is 0.

# MWE

The following example a pair of knobs for each of four different values: one with the default minimum value of 0, and another with minimum value equal to the value (which results in two identical knobs with values of 0 in the top row).

```python
from nicegui import ui

N = 4

for i in range(N):
    value = i / N
    with ui.row().classes("items-center"):
        for min_value in [0.0, value]:
            ui.knob(value=value, min=min_value, track_color="grey-2", show_value=True)
        ui.label(f"value = {value}")

ui.run()
```

# Expected behavior

The value should be showing in each knob.

# Actual behavior

The value does not show in either knob with value 0. However, since the value shows for both knobs for all the non-zero values, it seems that the value is hidden because it is 0, not because it is the minimum value.

I haven't tracked down the cause in NiceGUI source code, but this behavior seems consistent with the value being checked directly for truthiness, as in the following pseudocode:
```python
if value:
    show_value()
```

**Comment by falkoschindler** [comment]
Oh wow, thanks for reporting this bug, @AndrewMHenry!
It looks like it's even larger. The know creates a label that binds its text from the knob value. For a value "0" the label is indeed empty. But it also happens with other value elements:

```py
number = ui.number(value=0)
ui.label().bind_text_from(number, 'value')
```

The label is empty if the number is 0.

**Comment by falkoschindler** [comment]
And it even happens with binding from an arbitrary object (no value element needed):

```py
model = type('', (object,), {'value': 0})()
ui.label().bind_text_from(model, 'value')
ui.button('Set to 0', on_click=lambda: setattr(model, 'value', 0))
ui.button('Set to 1', on_click=lambda: setattr(model, 'value', 1))
```

**Comment by AndrewMHenry** [comment]
@falkoschindler Wow, that was fast!

Sure thing; I've found NiceGUI really useful and fun to work with, so thanks to you and the team for all your hard work!

**Comment by falkoschindler** [comment]
It's still getting better... 😄
```py
ui.label(0)  # not visible
ui.label(1)
```

**Comment by falkoschindler** [comment]
Here is the check that treats 0 as "no text":
https://github.com/zauberzeug/nicegui/blob/fa32577d52e29e5dadcaa8d386a54d7e9982476c/nicegui/templates/index.html#L107

**Comment by AndrewMHenry** [comment]
Aha, that looks promising!

**Comment by falkoschindler** [comment]
Ok, should be fixed via with commit [c4f18b6](https://github.com/zauberzeug/nicegui/commit/c4f18b6fc33e1a27b2ea132a2673642839764a03).

**Comment by AndrewMHenry** [comment]
I checked out that commit, and I can confirm it fixed my MWE on my end. Thanks again for the speedy response!
